test/casing.test.js

```diff
@@ -0,0 +1,62 @@
+const test = require('ava');
+const { snek, desnek } = require('../lib/casing.js');
+
+test('transform object keys from camelCase to snake_case', t => {
+	const input = {
+        foo: "foo",
+        fooBar: "foo_bar",
+        fooBar100: "foo_bar_100",
+        foo100bars: "foo_100bars",
+        foo200Bars: "foo_200_bars",
+        FooBar: "FooBar",
+        Foo: "Foo",
+        nestedObject: {
+            fooBar: "foo_bar",
+        },
+    };
+
+    const expected = {
+        foo: "foo",
+        foo_bar: "foo_bar",
+        foo_bar_100: "foo_bar_100",
+        foo_100bars: "foo_100bars",
+        foo_200_bars: "foo_200_bars",
+        FooBar: "FooBar",
```

Comment:
Wouldn't this want to be `foo_bar`? Or is it strict camel case to snake case, where only `fooBar` becomes `foo_bar`?

Reply:
Strict to camel case as it was already for long time, didn't change that at all.